styles.css

```diff
@@ -0,0 +1,111 @@
+body {
+  font-family: "Bebas Neue", sans-serif;
+  margin: auto;
+  font-size: 1.3rem;
+}
+
+h1 {
+  text-align: center;
+  font-size: 3rem;
+  background-color: #b1c5c3;
+  margin-top: 0;
+  padding: 10px;
+}
+
+.questionBox {
+  display: flex;
+  justify-content: center;
+  position: relative;
+  margin-bottom: 30px;
+}
+
+.img-bookmark-white {
+  height: 40px;
+  position: absolute;
+  top: -18px;
+  right: 45px;
+}
+
+.questionCard {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  text-align: center;
+  flex-wrap: wrap;
+  align-items: left;
+  width: 70vw;
+  height: 25vh;
+  border: 1px solid black;
+  border-radius: 10px;
+  margin: auto;
```

Comment:
instead of margin auto you can give a fixed padding to the container of the question cards.

Reply:
I replaced margin auto with margin-bottom (20px) to have a space between the cards (questions card and answer card which have both the question-card class. Is there a possibility to avoid margin-bottom?